mail.pohjoistech.fi.email.json

```diff
@@ -0,0 +1,57 @@
+{
+  "providerId": "mail.pohjoistech.fi",
+  "providerName": "PohjoisMail",
+  "serviceId": "email",
+  "serviceName": "Mail",
+  "version": 1,
+  "syncPubKeyDomain": "domainconnect.pohjoistech.fi",
+  "syncRedirectDomain": "mail.pohjoistech.fi",
+  "logoUrl": "https://r2.pohjoistech.fi/logo.png",
+  "description": "Enables a domain to work with PohjoisMail, a mail service offered by Pohjoistech.",
+  "variableDescription": "ip is the IP address of the server, and key is the DKIM public key.",
+  "records": [
+    {
+      "pointsTo": "%ip%",
+      "type": "A",
+      "host": "pohjoismail",
+      "essential": "OnApply",
+      "ttl": 14400
+    },
+    {
+      "pointsTo": "pohjoismail.%domain%",
```

Comment:
I'd rather expect `pohjoismail.%fqdn%.` if you want to CNAME to the host with A record